src/plots/cartesian/transition_axes.js

```diff
@@ -17,6 +17,8 @@ var Drawing = require('../../components/drawing');
 var Axes = require('./axes');
 var axisRegex = /((x|y)([2-9]|[1-9][0-9]+)?)axis$/;
 
+var LAST_TRANSLATION_RE = /translate\([^)]*\)\s*$/;
```

Comment:
No, for two reasons, I think. First, it doesn't get the last translation explicitly, and second, it teases apart the numbers involved, which is unnecessary here. We only need to preserve the last transform without modification, not parse and reconstruct it.